client/packages/common/src/intl/locales/es/replenishment.json

```diff
@@ -63,6 +63,7 @@
   "messages.deleted-shipments_other": "Se eliminaron {{count}} envíos",
   "messages.changing-max-mos": "Esto cambiará el objetivo máximo de meses de existencias.",
   "messages.changing-min-mos": "Esto cambiará el umbral mínimo de meses de existencias.",
+  "messages.unassign-min-mos": "Esto eliminará el umbral mínimo de meses de existencias. Los meses mínimos de stock serán por defecto los mismos que los meses máximos de stock.",
```

Comment:
This is my best guess at the translation. I wonder if I should flag this as unconfirmed? Or omit it and wait for an expert?

Reply:
You can leave translations out... someone will translate it eventually...

Reply:
Should I remove the translation strings which have been previously translated but are now out of date?
I can imagine that it would be easy to lose track of strings if they remain but are wrong.

For example if I change the english string, and now the french string is out of date. Should I remove the french string so that it is obvious that it needs to be retranslated?